content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -120,6 +121,75 @@ If the volume plugin does not support mount options but mount options are
 specified, provisioning will fail. Mount options are not validated on either
 the class or PV, so mount of the PV will simply fail if one is invalid.
 
+### Volume Binding Mode
+
+{{< feature-state for_k8s_version="v1.12" state="beta" >}}
+
+**Note:** This feature requires the `VolumeScheduling` feature gate to be
+enabled.
+
+The `volumeBindingMode` field controls when [volume binding and dynamic
+provisioning](/docs/concepts/storage/persistent-volumes/#provisioning) should occur.
+
+By default, the `Immediate` mode indicates that volume binding and dynamic
+provisioning occurs once the PVC is created. For storage
```

Comment:
Should we indicate here that PVC == PersistentVolumeClaim?

Reply:
It doesn't hurt to be explicit.

Reply:
PVC and PersistentVolumeClaim were introduced at the beginning of this page